xmtp_mls/src/groups/group_permissions.rs

```diff
@@ -324,14 +327,6 @@ impl PolicySet {
     }
 }
 
-fn default_add_installation_policy() -> MembershipPolicies {
-    MembershipPolicies::allow()
-}
-
-fn default_remove_installation_policy() -> MembershipPolicies {
-    MembershipPolicies::deny()
-}
```

Comment:
These seemed like a bit of needless indirection to only be called in the `new()` constructor above.